src/test/resources/features/sfdc/opportunities/opportunity.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Opportunities
+
+  Background:
+    Given I use the "sfdc" service and the "admin" account
+    And I send a "POST" request to "/sobjects/Opportunity" with json body
+    """
+    {
+    "Name": "Opportunity0002 created by cucumber",
+    "StageName": "Prospecting",
+    "CloseDate": "2019-11-02",
+    "Probability": "10",
+    "Amount": 2000
+    }
+    """
+    And I save the response as "O"
+
+  Scenario: PATCH Accounts
```

Comment:
Replace 'Opportunities' instead of 'Accounts'

Reply:
Thank you Benjamin it was updated